fs/src/commonMain/kotlin/com/suparnatural/core/fs/FileSystem.kt

```diff
@@ -67,6 +67,11 @@ expect object FileSystem {
      */
     fun readFile(pathComponent: PathComponent, encoding: ContentEncoding = ContentEncoding.Utf8): String?
 
+    /**
+     * Returns the contents of the file located at `path`as ByteArray.
+     */
+    fun readFileAsByteArray(path: String):ByteArray?
```

Comment:
1. Can you rename this to `readFile(path: String): ByteArray?` The return type is sufficient to mark that it reads as byte array.

2. Can you also add `readFile(path: PathComponent): ByteArray?` to keep it consistent with the rest of the API.